dev-db/pg-gvm/pg-gvm-22.4.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Greenbone Library for helper functions in PostgreSQL"
+HOMEPAGE="https://www.greenbone.net https://github.com/greenbone/pg-gvm"
+SRC_URI="https://github.com/greenbone/pg-gvm/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+# Tests requires a running database that match up with the current
+# testing slot. Won't run from ${ED}, want's to install too early.
+RESTRICT="test"
+
+DEPEND="
+	acct-group/gvm
+	acct-user/gvm
+	>=dev-libs/glib-2.42:2
+	>=dev-libs/libical-1.0.0
+	>=net-analyzer/gvm-libs-20.8
+"
+
+RDEPEND="
+	${DEPEND}
+	>=dev-db/postgresql-9.6[uuid]
+"
+
+BDEPEND="
+"
+src_prepare() {
+	cmake_src_prepare
```

Comment:
Do you really need to explicitly call the `cmake_src_*` functions? I believe you should be able to drop the explicit declaration of `src_{prepare,configure,compile,install}`.